### PR TITLE
Release v1.4.0 announcement

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: _posts/2017-01-05-elixir-v1-4-0-released.markdown [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Elixir v1.4 released
+author: José Valim
+category: Releases
+excerpt: Elixir v1.4 brings many improvements to the language, its standard library and the Mix build tool.
+---
+
+Elixir v1.4 brings new features, enhancements and bug fixes into Elixir. The most notable changes are the addition of the `Registry` module, the `Task.async_stream/3` and `Task.async_stream/5` function which aid developers in writing concurrent software, and the new application inference and commands added to Mix.
```
Comment (eksperimental) [inline _posts/2017-01-05-elixir-v1-4-0-released.markdown]: this is a bit repetitive,
I would go with something like.
> Elixir v1.4 brings new features, enhancements and bug fixes into the language.

or 

> Elixir v1.4 brings in new features, enhancements and bug fixes.

File: _posts/2017-01-05-elixir-v1-4-0-released.markdown [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+A registry may have unique or duplicate keys. Every key-value pair is associated to the process registering the key. Keys are automatically removed once the owner process terminates. Starting, registering and looking up keys is quite straight-forward:
+
+```iex
+iex> Registry.start_link(:unique, MyRegistry)
+iex> {:ok, _} = Registry.register(MyRegistry, "hello", 1)
+iex> Registry.lookup(MyRegistry, "hello")
+[{self(), 1}]
+```
+
+Finally, huge thanks to [Bram Verburg](https://twitter.com/voltonez) has performed [extensive benchmarks](https://docs.google.com/spreadsheets/d/1MByRZJMCnZ1wPiLhBEnSRRSuy1QXp8kr27PIOXO3qqg/edit#gid=0) on the registry to show it scales linearly with the number of cores by increasing the number of partitions.
```
Comment (eksperimental) [inline _posts/2017-01-05-elixir-v1-4-0-released.markdown]: **who** has performed

File: _posts/2017-01-05-elixir-v1-4-0-released.markdown [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+Simply running:
+
+    mix escript.install hex ex_doc
+
+will fetch `ex_doc` and its dependencies, build them, and then install `ex_doc` to `~/.mix/escripts` (by default). After adding `~/.mix/escripts` to your `PATH`, running `ex_doc` is as simple as:
+
+    ex_doc
+
+You can now also install archives from Hex in this way. Since they are fetched and built on the user's machine, they do not have the same limitations as pre-built archives. However, keep in mind archives are loaded on every Mix command and may conflict with modules or dependnecies in your projects. For this reason, escripts is the preferred format for sharing executables.
```
Comment (fishcakez) [inline _posts/2017-01-05-elixir-v1-4-0-released.markdown]: "dependnecies"

**Pull request merged**